internal/exec/stages/disks/disks.go

```diff
@@ -212,7 +212,7 @@ func (s stage) createRaids(config types.Config) error {
 			args = append(args, util.DeviceAlias(string(dev)))
 		}
 
-		if err := s.Logger.LogCmd(
+		if _, err := s.Logger.LogCmd(
```

Comment:
Can you break that into a separate commit? The intent of commits is to collect groups of changes into a single, logical patch. If there is too much going on in each of them, it's very difficult to follow and manipulate the history.